functions/index.js

```diff
@@ -60,6 +60,57 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
     });
 });
 
+exports.createNewSprint = functions.https.onRequest((request, response) => {
+    cors(request, response, () => {
+        console.log(request);
+        var title = request.body.data.Title;
+        var des = request.body.data.Description;
+        var status = request.body.data.Status;
+        var totalDevelopmentTask = request.body.data.development;
+        var totalBusinessTask = request.body.data.business;
+        var totalMarketingTask = request.body.data.marketing;
+        var lastSprintId = 0;
+        var newtSprintId = lastSprintId + 1;
+        var currentSprintId = "S" + newtSprintId;
+        var totalTask = totalDevelopmentTask + totalBusinessTask + totalMarketingTask;
+        var date = new Date();
+        var startDate = date.getDate() + "/" + (date.getMonth() + 1) + "/" + date.getFullYear();
+        var date = new Date();
```

Comment:
Why do we have two dates. very illogical code. Please use logic.

Reply:
Changed